journal/src/test/java/io/zeebe/journal/file/JournalTest.java

```diff
@@ -473,6 +474,29 @@ void shouldNotReadDeletedEntries() {
     assertThat(reader.hasNext()).isFalse();
   }
 
+  @Test
+  public void shouldInvalidateAllEntries() throws Exception {
```

Comment:
What is this test verifying?

Reply:
That all entries are invalidated. If we delete 2 entries, write one of same length as the previous one then the second deleted entry should still not be read because all entries are invalidated and not just the one after the truncation index